public/scss/buttons.scss

```diff
@@ -6,12 +6,23 @@
     font-size: 1.1em;
     font-weight: bolder;
     letter-spacing: 1px;
-    padding: 0.9em 0;
+    padding: 0.9em 2em;
+    margin: 10px 0;
     width: 100%;
 
     &[disabled]:hover {
         color: $blue-light;
     }
+
+    &.center-block {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    @media (min-width: $tablet-min) {
+        width: auto;
```

Comment:
Haha actually that was what I was going for with desktop and tablet. To me it looked funky having the button grow super large. It stops looking like a button when it gets too wide. Thoughts on an alternative design for this sized screen?